### ISSUE TITLE
TTV Fitting With Jaxoplanet?

### ISSUE THREAD

**Issue by TylerFair** [exoplanet-dev/jaxoplanet#233]
Thanks for the updated documentation regarding jaxoplanet lately! I will preface by saying I am a relative beginner to jax+jaxoplanet but have experience with pymc5+exoplanet. I was just checking if jaxoplanet supported any transit timing fitting analogous to exoplanet's TTVOrbit? I cannot find anything similar so was not sure if this is because the functionality has not been added yet/there are other ways to solve this using jaxoplanet or if pymc does a "better job" for this problem. 

The alternative solution is obviously using TransitModel while fitting for each transit time, but I have typically avoided this in the past due to the high independent random variates this creates (i.e. tt1, tt2, ttN rather than one tt with dim=N). Is there an intended vectorization approach such as using jax.vmap over all transit times? 

Many thanks,
Tyler

**Comment by lgrcia** [comment]
Hi @TylerFair and thanks for raising this point! The functionality has not been added yet. I would have a look to implementing that asap!

**Comment by TylerFair** [comment]
Hi @lgrcia , I have given a probably novice attempt at this myself with some spare side-project time, as mentioned in the PR, it probably could be improved with further jax operations but *hopefully* this can be implemented in the meantime to help any new users with TTV systems, I have found no significant problems with it thusfar from standard implementation.

**Comment by lgrcia** [comment]
Thanks @TylerFair, I'll have a close look at #251 and chat about details over there then!

**Comment by TylerFair** [comment]
Closing this issue as solved, thanks!
